### PR TITLE
Replace BaseCommand.option_list with parser.add_argument

### PR DESCRIPTION
**Pull request by dongbohu**
This PR addresses #3. It also adds `.gitignore` file.

### CODE REVIEW

**Comment by dongbohu** [comment]
I have tested the code on my local desktop.

File: organisms/management/commands/organisms_create_or_update.py [hunk 1]
```diff
@@ -25,19 +24,26 @@
...
-            help="Organism common name, e.g. 'Human'"),
-        make_option(
-            '--scientific_name', action='store', dest='scientific_name',
-            help="Organism scientific/binomial name, e.g. 'Homo sapiens'"),
-    )
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--taxonomy_id',
+            type=int,
+            dest='taxonomy_id'
```
Comment (dhimmel) [inline organisms/management/commands/organisms_create_or_update.py:31]: Checking that you're aware dest is redundant here.

File: organisms/management/commands/organisms_create_or_update.py [hunk 2]
```diff
@@ -25,19 +24,26 @@
...
+        )
+
+        parser.add_argument(
+            '--common_name',
+            type=str,
```
Comment (dhimmel) [inline organisms/management/commands/organisms_create_or_update.py]: str is default

**Review by dhimmel — COMMENTED** [review]
Some opportunities to remove arguments that are defaults, should you feel so inclined.

File: organisms/management/commands/organisms_create_or_update.py (hunk 2 above)
Comment (dongbohu) [reply organisms/management/commands/organisms_create_or_update.py]: :+1:

File: organisms/management/commands/organisms_create_or_update.py (hunk 1 above)
Comment (dongbohu) [reply organisms/management/commands/organisms_create_or_update.py:31]: Thanks. I do want to specify `dest` explicitly here.

**Comment by dhimmel** [comment]
Got it. Approving PR.

One other note is that --taxonomy-id with hypen not underscore is perhaps a
bit more pythonic but not worth breaking backwards compatibility IMO

**Pull request merged**